internal/core/items/text.rs

```diff
@@ -158,29 +158,40 @@ impl RenderText for ComplexText {
     }
 
     fn font_request(self: Pin<&Self>, self_rc: &ItemRc) -> FontRequest {
```

Comment:
many large duplicated font_request functions. I wonder if this couldn't be a helper function

`font_request(self_rc, self.font_family(), self.font_weight(), self.font_size(), self.letter_spacing(), self.italic())`


Not sure if worth it though